### ISSUE TITLE
Issue with cidr block 172.128.0.0/9 

### ISSUE THREAD

**Issue by ShiriMoran** [np-guard/vpc-network-config-analyzer#535]
In branch `471_explain_all_to_all`  we execute explainability all-to-all and in testing `TestMultiExplainability` get the following errors, all related to the above cidr. The errors seems non-related to explainability.

```

62. could not find containing config node for Public Internet [172.128.0.0/9]
63. could not find containing config node for Public Internet [172.128.0.0/9]
81. isIngress: false , target 172.128.0.0/9, subnetCidr: 10.240.1.0/24, inSubentCidr 10.240.1.4, could not find connectivity for given target + inSubentCidr
82. isIngress: true , target 172.128.0.0/9, subnetCidr: 10.240.1.0/24, inSubentCidr 10.240.1.4, could not find connectivity for given target + inSubentCidr
164. could not find containing config node for Public Internet [172.128.0.0/9]
165. could not find containing config node for Public Internet [172.128.0.0/9]
181. could not find containing config node for Public Internet [172.128.0.0/9]
182. could not find containing config node for Public Internet [172.128.0.0/9]

```

**Comment by adisos** [comment]
Seems that the issue is in that branch , within funcion `CreateMultiExplanationsInput`.
It does not necessarily create input for explainability (and connectivity analysis) with disjoint ip-blocks. 
(message "...could not find connectivity for given target + inSubentCid" indicates the input for nacl analysis had cidr which is not from expected disjoint ip-blocks). 

printing the input pairs it generates, I see it includes the two following:
```
- vpc: test-vpc0-ky, src: vsi1-ky[10.240.1.4], dst: Public Internet 172.217.22.46/32
- vpc: test-vpc0-ky, src: vsi1-ky[10.240.1.4], dst: Public Internet (all ranges)
```

though the second should be only part from Public Internet , the one without 172.217.22.46/32. 

from the allowed connectivity report, we have in that VPC:
```
vsi1-ky[10.240.1.4] => Public Internet 172.217.22.46/32 : All Connections
```

So the current code to generate input pairs for multi-explainability has issues.
Should cosnider handling external nodes more carefully, use disjoint ip-blocks, and per config separately.

**Comment by ShiriMoran** [comment]
Not sure the issue and the fix is indeed in `CreateMultiExplanationsInput`.
In fact, explainability is robust to inputs and the `CreateMultiExplanationsInput` merely creates inputs. 
The fix should be in the way this input is handled. I just need to handle this input (which is @haim-kermany 's responsibility) more carefully (my responsibility). I already have code that does it, just needs to use it.

**Comment by ShiriMoran** [comment]
fixed by PR #536
